packages/translate-results/lib/startup.ts

```diff
@@ -0,0 +1,136 @@
+import {Translation} from './types';
+import t from './texts';
+
+/**
+ * Translate the reason for test not being executed
+ */
+export function translateStartupError(code: StartupError): Translation {
```

Comment:
Perhaps "error" does not fit that well here. Some of those are actually recoverable.

Reply:
translateStartupIssues?